apps/scheduler/src/imbi/scheduler/identity.py

```
@@ -0,0 +1,234 @@
+
+import httpx
+
+from imbi.scheduler import models, settings
+
+LOGGER = logging.getLogger(__name__)
+
+#: Refresh a service-account token this long before it expires, so a run never
+#: starts with a credential about to lapse mid-flight.
+REFRESH_MARGIN = datetime.timedelta(seconds=60)
+
+TOKEN_PATH = '/auth/token'
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "TOKEN_PATH"

(S105)